src/makeRobotAccountant.js

```diff
@@ -7,6 +7,26 @@
 
 function makeRobotAccountant() {
   // write code here
+  // random comment cuz i forgot to switch branch and need to make changes :P
+  let counter = 0;
+
+  function getSum(a) {
+    function bullshitNesting(b) {
+      if (counter > 2 && counter % 2 === 1) {
+        counter++;
+
+        return `Bzzz... Error!`;
+      } else {
+        counter++;
+
+        return a + b;
+      }
+    }
+
+    return bullshitNesting;
+  }
```

Comment:
You can just use arrow functions `() => {}` instead off naming your functions `bullshitNesting`. Please remember to avoid such Easter eggs.